README.md

```diff
@@ -25,10 +25,8 @@ git clone https://github.com/conan-io/cmake-conan.git -b develop2
 This repository contains a `CMakeLists.txt` with an example project that depends on `fmt`. 
 
 ```bash
-cd cmake-conan
```

Comment:
The ``cd cmake-conan`` is still necessary, we are not yet inside the cloned project